backend/api/swagger.yml

```diff
@@ -454,35 +445,18 @@ components:
         uuid:
           type: string
           description: UUID of the user in the database
-          example: "u-b49358fc-7137-11ea-bd4f-e470b8b61683"
-        user_name:
-          type: string
-          description: user name of the user
-          example: david01235
-        role:
+          example: "b49358fc-7137-11ea-bd4f-e470b8b61683"
+        name:
           type: string
-          description: what the user does in this system
-          example: "navigator"
+          description: name of the user in form "first last"
+          example: David Goldstein
 
     User:
       type: object
       properties:
         id:
           type: string
-          description: uuid of the user starting with "u-"
-          example: "u-sdf23f-f23joif-sfdh23-sefue"
-        user_name:
-          type: string
-          description: unique username of the user
-          example: "david2134"
-        role:
-          type: string
-          description: what the user does in the system
-          example: "navigator"
-        is_admin:
-          type: boolean
-          description: is the user an admin in system?
-          example: true
+          description: UNDER CONSTRUCTION
```

Comment:
why revert this change? It's nice to have the /auth endpoints fleshed out in the API now that we support them

Reply:
Right, sorry that was accidental. Restores